core/src/main/java/com/yahoo/oak/Block.java

```diff
@@ -14,14 +14,7 @@
 
 class Block {
 
-    /*
-    For performance reasons, we keep two ByteBuffer objects that refers to the same buffer.
-    writeBuffer is the original allocated buffer, and readBuffer is a duplicated version of writeBuffer
-    but with read-only access permissions.
-    This allow the user to supply the a read-only ByteBuffer (when needed) without instantiating a new object.
-     */
-    private final ByteBuffer readBuffer;
-    private final ByteBuffer writeBuffer;
```

Comment:
Just wonder now... Many sketches may reside on the same block, it means they all have the same hash code. How will this sketches mapping work?

Reply:
Sketches use both the object-ID of the buffer and its position